File: src/main/java/module/opponentspy/OpponentTeam/PlayedPosition.java
package module.opponentspy.OpponentTeam;

import core.model.match.MatchType;

public class PlayedPosition {
	
	public PlayedPosition(int positionId, byte tacticId, MatchType matchType, double ratingStart, double ratingEnd) {
		this.positionId = positionId;
		this.tacticId = tacticId;
		this.matchType = matchType;
		this.ratingEnd = ratingEnd;
		this.ratingStart = ratingStart;
	}
	
	public int positionId;
	public byte tacticId;
	public MatchType matchType;
	public double ratingStart;
	public double ratingEnd;
}

File: src/main/java/module/opponentspy/OpponentTeam/PlayedPosition.kt
package module.opponentspy.OpponentTeam;

import core.model.match.IMatchType;

public class PlayedPosition {
	
	public PlayedPosition(int positionId, byte tacticId, IMatchType matchType, double ratingStart, double ratingEnd) {
		this.positionId = positionId;
		this.tacticId = tacticId;
		this.matchType = matchType;
		this.ratingEnd = ratingEnd;
		this.ratingStart = ratingStart;
	}
	
	public int positionId;
	public byte tacticId;
	public IMatchType matchType;
	public double ratingStart;
	public double ratingEnd;
}
